src/components/common/StyledSelect/components/CustomIndicatorsContainer.tsx

```diff
@@ -0,0 +1,43 @@
+import { IconButton, theme, useClipboard, useColorModeValue } from "@chakra-ui/react"
+import { transparentize } from "@chakra-ui/theme-tools"
+import { GroupBase, IndicatorsContainerProps, components } from "chakra-react-select"
+import { Check, Copy } from "phosphor-react"
+import parseFromObject from "utils/parseFromObject"
+import { StyledSelectProps } from "../StyledSelect"
+
+const CustomIndicatorsContainer = ({
+  children,
+  value,
+  ...rest
+}: {
+  value: StyledSelectProps["value"]
+  pathToCopy: string
+} & IndicatorsContainerProps<unknown, boolean, GroupBase<unknown>>) => {
+  const successIconColor = useColorModeValue("green.800", "green.200")
+
+  const { onCopy, hasCopied } = useClipboard(parseFromObject(value, "value"), 1000)
+
+  return (
+    <components.IndicatorsContainer {...rest}>
+      {value && (
+        <IconButton
+          backgroundColor={
+            hasCopied && transparentize(successIconColor, 0.12)(theme)
+          }
+          color={hasCopied && successIconColor}
+          icon={hasCopied ? <Check /> : <Copy />}
+          aria-label={"Copy"}
+          size={"xs"}
+          rounded={"full"}
+          variant={"ghost"}
+          onMouseDown={() => {
```

Comment:
maybe `onClick` + `e.stopPropagation()` would work properly and then we wouldn't need to use `onMouseDown` here ;) Please check and lmk!

Reply:
I found that react-select uses `onMouseDown` to set the dropdown state, which happens before `onClick`. So we kind of need to get ahead of it by using onMouseDown here as well

Reply:
Hmm, I see, thanks for the explanation.
The problem with the current solution is that it won't work on mobile (or at least I couldn't really click on the copy button), so let's brainstorm about this issue on Discord ;)